tests/e2e/normalize.go

```diff
@@ -99,6 +101,18 @@ func normalizeKRMObject(t *testing.T, u *unstructured.Unstructured, project test
 	// Specific to Dataflow
 	visitor.sortAndDeduplicateSlices.Insert(".spec.additionalExperiments")
 
+	// Specific to Dataproc
+	{
+		visitor.ReplacePath(".status.clusterUuid", "${clusterUuid}")
+		visitor.ReplacePath(".status.status.stateStartTime", PlaceholderTimestamp)
+		visitor.ReplacePath(".status.statusHistory[].stateStartTime", PlaceholderTimestamp)
+
+		visitor.ReplacePath(".status.metrics.hdfsMetrics.dfs-capacity-present", "56789")
+		visitor.ReplacePath(".status.metrics.hdfsMetrics.dfs-capacity-remaining", "56789")
+		visitor.ReplacePath(".status.metrics.hdfsMetrics.dfs-capacity-total", "56789")
+		visitor.ReplacePath(".status.metrics.hdfsMetrics.dfs-capacity-used", "56789")
+	}
+
```

Comment:
Or move this to dataproc?

Reply:
So that would be mockgcp/mockdataproc.  This is specific to the KRM form, so it _technically_ doesn't belong there.  That said, it would definitely be convenient to group these by KRM group, and it wouldn't be too bad to put it into the mockgcp: mockgcp is meant for KCC, after all...

Maybe we try it in another PR?